content/root/quick-start-guide.textile

```diff
@@ -87,70 +108,104 @@ blang[java].
   compile 'io.ably:ably-android:1.2.0'
   ```
 
-  In the above example a specific version of the library is referenced, however we recommend you check which is the latest stable version and always use that. Follow links to get the latest stable release for "Java":https://bintray.com/ably-io/ably/ably-java and "Android":https://bintray.com/ably-io/ably/ably-android.
+  *Note*: In the above examples a specific version of the SDK is referenced. It's recommended to always use the latest stable version available for "Java":https://bintray.com/ably-io/ably/ably-java and "Android":https://bintray.com/ably-io/ably/ably-android.
```

Comment:
Ably recommends that you always use? (Active voice, clear about who's recommending this)

Reply:
Done.